src/tablist.js

```diff
@@ -332,8 +341,15 @@ SideTabList.prototype = {
     query = normalizeStr(query);
     let notShown = 0;
     for (let tab of this.tabs.values()) {
-      const show = normalizeStr(tab.url).includes(query) ||
-                   normalizeStr(tab.title).includes(query);
+      let show;
+      console.dir(this.useRegExpFilter);
+      if (this.useRegExpFilter) {
+        show = normalizeStr(tab.url).match(query) ||
```

Comment:
You should factorize the normalized strings at least (you could even factorize the comparison too)